From fd0479bbd8a3f6ea85b29aa6145d31eee5372c20 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Tue, 7 Jan 2025 11:59:41 +0100
Subject: [PATCH 1424/1695] plat-stm32mp1: conf: default enable CFG_STM32_LTDC

Enable always the display controller to set/unset the secure layer.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 97eaf642a876930131857ab6d40bb3a23a64a1e8 plat-stm32mp1: conf: default enable CFG_STM32_LTDC]
Change-Id: Ic0b8fafb9550d15c6a428db5d0c914f4b700ad47
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/427229
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/plat-stm32mp1/conf.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index 8704bf786..9a885282a 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -272,7 +272,6 @@ endif
 ifeq ($(CFG_WITH_TUI),y)
 $(call force,CFG_DISPLAY,y,Mandated by CFG_WITH_TUI)
 $(call force,CFG_FRAME_BUFFER,y,Mandated by CFG_WITH_TUI)
-$(call force,CFG_STM32_LTDC,y,Mandated by CFG_WITH_TUI)
 # Provision virtual space to fit 10MByte plus the TUI frame buffer
 CFG_TUI_FRAME_BUFFER_SIZE_MAX ?= 0x01000000
 CFG_RESERVED_VASPACE_SIZE ?= (10 * 1024 * 1024 + $(CFG_TUI_FRAME_BUFFER_SIZE_MAX))
@@ -318,6 +317,7 @@ CFG_STM32_HASH ?= y
 CFG_STM32_I2C ?= y
 CFG_STM32_IWDG ?= y
 CFG_STM32_LPTIMER ?= y
+CFG_STM32_LTDC ?= y
 CFG_STM32_PKA ?= y
 CFG_STM32_PWR_IRQ ?= y
 CFG_STM32_RNG ?= y
-- 
2.47.2

